discover/controller_test.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2017 Istio Authors
```

Comment:
remove this, see comment on controller.go

Reply:
done